diagnostic.md

```diff
@@ -14,7 +17,16 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+```js
```

Comment:
Please try to answer this on your own, rather than simply copying the solution code